detox/android/detox/src/testFull/java/com/wix/detox/common/ReflectUtilsTest.kt

```diff
@@ -0,0 +1,14 @@
+package com.wix.detox.common
+
+import com.wix.detox.espresso.action.common.ReflectUtils
+import org.junit.Test
+import org.assertj.core.api.Assertions.assertThat
+
+private const val TEST_CLASS = "com.facebook.react.views.slider.ReactSlider"
+
+class ReflectUtilsTest {
```

Comment:
Nice!

1. Could you Spek it? Or do you think we shouldn't?
2. Could you also check for the `return true` case?